custom/icds_reports/utils/aggregation_helpers/tests/sql_output/agg-awc.distributed.txt

```diff
@@ -316,6 +320,15 @@
                   WHEN awc_building = 4 THEN 'partial_covered_space'
                 ELSE NULL END AS infra_type_of_building,
                 CASE WHEN source_drinking_water IN (1, 2, 3) THEN 1 ELSE 0 END AS infra_clean_water,
+                toilet_facility,
+                CASE
+                  WHEN type_toilet = 1 THEN 'Pit type (Latrine)'
+                  WHEN type_toilet = 2 THEN 'Only urinal'
+                  WHEN type_toilet = 3 THEN 'Flush system'
+                  WHEN type_toilet = 4 THEN 'Other'
+                ELSE NULL END AS type_toilet,
```

Comment:
When/how do we make the decision to store a string versus storing some enumeration/slug versus storing an integer?

I think there was some point in time where there was talk of translating the dashboard which (to me) would have meant not to keep plain english text in the database, but instead implement any translation from integer to text in the UI layer

Reply:
I was not aware of that talk though. But makes sense to me though. I made this decision because of two reasons 1) I did not wanted it to again interpret int numbers to text and did not want different usage to  interpret differently by mistake. 2) Another column in this table similar is already happening.

But it sounds a valid discussion here. I see some valid points to store as ints and parse on UI:
1) easy translation
2) change in text wont require data correction in agg tables.

@calellowitz  whats your thoughts on this.

Reply:
Strongly agree that these should be stored as ints. That was the only review comment i was going to make so will just leave that here